main/src/nbmain.cxx

```diff
@@ -43,10 +43,10 @@ using namespace std;
 
 #ifdef WIN32
 #include <process.h>
-static string pathsep("\\");
+std::string pathsep = "\\";
```

Comment:
Why remote the static keyword?  ... humm right it has no practical consequence until the .o file is put in a library.